dockerfiles/github-com/Dockerfile

```diff
@@ -22,6 +22,7 @@ COPY --from=base /home/node/.npm-global /home/node/.npm-global
 
 COPY --from=base /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
 
+USER root
 # Removing debug@0.7.4 (setheader transitive) to satisfy some scanners still reporting this false positive
 RUN rm -rf /home/node/.npm-global/lib/node_modules/snyk-broker/node_modules/setheader/node_modules/debug
```

Comment:
@aarlaud, do you think it's worth to move it to the first stage after `RUN npm install --global snyk-broker`?